tests/openssl/enc/enc.c

```diff
@@ -26,10 +26,6 @@ int enc_test(int argc, char** argv, char** env)
     /* Directly use environ from host. */
     __environ = env;
 
```

Comment:
why was this removed?

Reply:
This was moved to enclave/crypto/openssl/init.c! When I made the original openssl_3 change I wanted to avoid making changes to enclave/crypto yet.